### DISCUSSION TITLE
分别监听 IPv4 和 IPv6 地址

### DISCUSSION THREAD

**Discussion by Someone-Practice** [XTLS/Xray-core#3063] [Q&A] [answered]
我可以有两个入站分别侦听两种地址类型吗？
例如。一个监听 0.0.0.0，另一个监听 [::]

**Comment by pulsarice** [comment, marked as answer]
I couldn't use 0.0.0.0 to only listen on IPv4. therefore I explicitly used my server's IPv4 as the listening address. I made another one to listen on IPv6 and now I have two inbounds on the same port on different IP stacks.
